diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/BlockStructureBracesWarnTest.kt

```diff
@@ -187,18 +187,6 @@ class BlockStructureBracesWarnTest {
         )
     }
 
-    @Test
-    fun `check empty fun bit with configuration`(){
```

Comment:
You can keep this test as an example that `BlockStructureBraces` doesn't check empty blocks.